python/tests/conftest.py

```diff
@@ -202,6 +218,29 @@ def _version_store_factory_impl(
     return out
 
 
+def lmdb_version_store_cleanup(version_store_fixture):
```

Comment:
Some docs on how this should be used? I guess it's sensitive to the ordering with `@pytest.fixture`?